velox/exec/tests/PlanBuilderTest.cpp

```diff
@@ -250,4 +253,50 @@ TEST_F(PlanBuilderTest, missingOutputType) {
       "outputType must be specified");
 }
 
+TEST_F(PlanBuilderTest, projectExpressions) {
+  // Non-typed Expression
+  auto data = ROW({"c0"}, {BIGINT()});
+  VELOX_CHECK_EQ(
+      PlanBuilder()
+          .tableScan("tmp", data)
+          .projectExpressions(
+              {std::make_shared<core::FieldAccessExpr>("c0", std::nullopt)})
+          .planNode()
+          ->toString(true, false),
+      "-- Project[1][expressions: (c0:BIGINT, ROW[\"c0\"])] -> c0:BIGINT\n"); // simple field access
```

Comment:
nit: Move comment to above the code as a separate line. E.g., 
```
// Non-typed Expression. Simple field access.
auto data = ROW({"c0"}, {BIGINT()});
...
```

Same for the comment at line 279 below.